### PR TITLE
DX-360 API: Intercept - Intercept signature and args are difficult to reason about

### PR DESCRIPTION
**Pull request by mccataldo**
closes DX-360

### CODE REVIEW

File: content/api/commands/intercept.md [hunk 1]
```diff
@@ -29,20 +29,30 @@ All intercepts are automatically cleared before every test.
 cy.intercept(url)
 cy.intercept(method, url)
 cy.intercept(routeMatcher)
+```
+
+See arguments [url](/api/commands/intercept#url-String-Glob-RegExp), [method](/api/commands/intercept#method-String) and [routeMatcher](/api/commands/intercept#routeMatcher-RouteMatcher)
```
Comment (cowboy) [inline content/api/commands/intercept.md:34]: The `url` link doesn't work, it looks like there's a missing blank line after `</Alert>` on line 95.

**Review by cowboy — CHANGES_REQUESTED** [review]
Look good other than one minor issue.

File: content/api/commands/intercept.md (hunk 1 above)
Comment (mccataldo) [reply content/api/commands/intercept.md:34]: OK try now

Comment (cowboy) [reply content/api/commands/intercept.md:34]: Awesome

**Review by cowboy — APPROVED** [review]

**Pull request merged**